### ISSUE TITLE
Slightly redefining the palette

### ISSUE THREAD

**Issue by jeromegn** [gratipay/gratipay.com#1195]
I'd like to redefine the colour palette, just a little bit. Basically adjusting the hues of the green and brown and then adding other colours for more flexibility.

![screen shot 2013-07-20 at 11 48 42 am](https://f.cloud.github.com/assets/43325/830471/f0833c6c-f153-11e2-9e41-4c5166052714.png)

Available here: http://www.colourlovers.com/palette/2403225/Family_Room

Green: #339966
Brown: #8A4B21

Basically, more vibrant colours and more choices.

Luckily the way our CSS is done, we can achieve that relatively easily.
